### ISSUE TITLE
Beatmap info wedge doesn't update BPM with time-ramp mods

### ISSUE THREAD

**Issue by sh0ckR6** [ppy/osu#14688]
**Describe the bug:** 
After selecting a time-ramping mod (ex. Wind Up, Wind Down) the displayed BPM on the beatmap widget does not update accordingly.

This could be shown as a range BPM range much like maps with multiple BPMs (shown below)
The range would also be shown on maps with one BPM, with the range being between original BPM and final BPM after ramp

**Screenshots or videos showing encountered issue:** 
##### BPM display without any time-ramp mods
![osu_2021-09-08_16-09-49](https://user-images.githubusercontent.com/68568464/132579242-0ed8ef77-8b68-4d00-96b2-9b10abb7ab9c.jpg)
##### BPM display with Wind Up (current release)
![osu_2021-09-08_16-09-57](https://user-images.githubusercontent.com/68568464/132579254-f615fb4b-133d-47db-99f9-f2105ff222ba.jpg)
##### BPM display with Wind Up (suggested solution)
![osu_2021-09-08_16-17-04](https://user-images.githubusercontent.com/68568464/132579261-53d56404-bceb-4eac-abd0-f2ff569e2d3a.jpg)

**osu!lazer version:** 
2021.907.0

**Logs:**
N/A but can provide if needed

---
I would be more than happy to resolve this issue myself, I just wanted to submit it to get it logged in the repo

**Comment by bdach** [comment]
this was known at the time. in general this calculation is not trivial to get 100% correct. any such calculation has to handle variable BPM/multiple timing points correctly.

**Comment by bdach** [comment]
also arguably a duplicate of #10580.

**Comment by sh0ckR6** [comment]
Considering the changes to BPM display for standard BPM shifting mods (DT/HT, etc.) were merged with mention of #10580, I would consider this issue a duplicate. I tried searching for duplicates of this issue but couldn't find any before opening this, but I never fully considered this to be a duplicate of #10580 until now. I'll close this issue for now unless it's decided that this is worth having it's own issue open for.
